zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1132,6 +1132,30 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         );
     }
 
+    pub(crate) fn call_context_lookup_write_with_counter(
```

Comment:
suggest to add comment to explain that rwc doesn't need  +1. Like your comment [here](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1719/files#diff-85bb226621e44af9ca0cd097278bcc4aa9c3aeaa80de587a962ed99196bbf7dcR136)

Reply:
Done and add comments :)